src/components/views/settings/tabs/user/SecurityUserSettingsTab.js

```diff
@@ -341,8 +342,9 @@ export default class SecurityUserSettingsTab extends React.Component {
         }
 
         let privacySection;
-        if (Analytics.canEnable() || CountlyAnalytics.instance.canEnable()) {
-            privacySection = <React.Fragment>
+        if ((Analytics.canEnable() || CountlyAnalytics.instance.canEnable()) && SettingsStore.getValue(UIFeature.ShowShareAnalyticsInformation)) {
+            privacySection =
```

Comment:
Please revert the whitespace change, to minimise changes to matrix master

Reply:
I have removed this flag entirely. I also updated user story documentation and feature documentations. While looking into other conditions embedded in "if", I can see that it looks for piwik, if it cant find then it wont show view in UI so maybe worth removing it.